### PR TITLE
PLATUI-1190: Build a webjar

### PR DESCRIPTION
**Pull request by matthewmascord**
This PR relates to https://github.com/hmrc/play-frontend-hmrc/pull/81

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## Changed
```
Comment (JoPintoPaul) [inline CHANGELOG.md:10]: Missing the version number here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -214,3 +214,49 @@ All work on GOV.UK should [use progressive enhancement](https://www.gov.uk/servi
...
+To run the code style checks independently you can run `npm run lint`.
+
+Automatic fixing is available for a number of the code style rules we apply.  To let the tool
+automatically fix the issues it can fix run `npm run lint:fix` - that will show you any remaining
+issues after the tool has fixed what it can for you.
+
+### WebJar publishing
+
+In order to make the hmrc-frontend assets easy to consume in a Play framework-based environment,
+the npm `build:webjar` task builds a Play-compatible webjar. This webjar is published to HMRC's open
```
Comment (JoPintoPaul) [inline CONTRIBUTING.md]: Maybe Scala or sbt compatible, rather than Play compatible.

File: CHANGELOG.md (hunk 1 above)
Comment (matthewmascord) [reply CHANGELOG.md:10]: This is deliberate, can't release until Jenkins is updated.

File: CONTRIBUTING.md (hunk 2 above)
Comment (matthewmascord) [reply CONTRIBUTING.md]: Updated to say JVM compatible, as webjars aren't specific to Scala/Play.

**Comment by matthewmascord** [comment]
retest this please

**Review by JoPintoPaul — APPROVED** [review]

**Pull request merged**